content/master/concepts/usages.md

```diff
@@ -189,6 +189,41 @@ spec:
           baz: qux
 ```
 
+### Replay blocked deletion attempt
+
+By default, the deletion of a `Usage` resource doesn't trigger the deletion of
+the resource in use even if there were deletion attempts blocked by the `Usage`.
+Replaying the blocked deletion is possible by setting the
```

Comment:
Usage protects by blocking the deletion of the used resource as long as there are using resources.

Unfortunately, Kubernetes garbage collection backoffs up to 15 mins and there seems to be [no way to reset](https://github.com/kubernetes/enhancements/pull/2840#issuecomment-1945499956) this backoff.

"replaying" is sending a delete request to the used resource when the Usage is gone iff there was a blocked delete request.

Reply:
I'd suggest an edit to be more explicit about that here, something like 

> Use {{<hover label="replay" line="6">}}replayDeletion: true{{</hover>}} to immediately create send a deletion request to Kubernetes instead of waiting for the Kubernetes backoff.

Feel free to reword, I'd just make it more obvious that "doing this makes deletion happen faster"